### PR TITLE
Deploy libappimage headers

### LINKED ISSUE #2: LibAppImage headers not being deployed.

### PR DESCRIPTION
**Pull request by azubieta**
fix #2

### CODE REVIEW

File: src/libappimage/CMakeLists.txt [hunk 1]
```diff
@@ -1,16 +1,20 @@
 cmake_minimum_required(VERSION 3.2)
 
-set(libappimage_public_header ${PROJECT_SOURCE_DIR}/include/appimage/appimage.h)
+set(libappimage_public_headers_list
+        ${PROJECT_SOURCE_DIR}/include/appimage/appimage.h
+        ${PROJECT_SOURCE_DIR}/include/appimage/appimage_shared.h)
```
Comment (TheAssassin) [inline src/libappimage/CMakeLists.txt]: `libappimage_shared`'s header should be installed in its own directory. This is breaking the separation.

File: src/libappimage/CMakeLists.txt [hunk 2]
```diff
@@ -1,16 +1,20 @@
...
 
 # both libraries use the same set of source files
 set(libappimage_sources
-    ${libappimage_public_header}
-    libappimage.c
-)
+        ${lipappimage_public_headers_list}
```
Comment (TheAssassin) [inline src/libappimage/CMakeLists.txt]: Don't change the indentation.

File: src/libappimage/CMakeLists.txt [hunk 3]
```diff
@@ -1,16 +1,20 @@
...
+        libappimage.c
+        )
```
Comment (TheAssassin) [inline src/libappimage/CMakeLists.txt]: Weird syntax. Please revert.

File: src/libappimage/CMakeLists.txt (hunk 1 above)
Comment (azubieta) [reply src/libappimage/CMakeLists.txt]: Actually, we agreed on putting all appimage related headers into the appimage dir. So I guess it right place.

**Comment by TheAssassin** [comment]
Fixed in https://github.com/AppImage/libappimage/commit/df5661c42991913a1952a071af9304d2e724cec0.

**Pull request closed without merging**